src/common/util/select-unit.ts

```diff
@@ -82,6 +112,20 @@ export function selectUnit(
   };
 }
 
+function getFirstDayOfTheWeek(date: Date, firstWeekday: number) {
```

Comment:
https://date-fns.org/v2.29.3/docs/startOfWeek

`startOfWeek(date, { weekStartsOn: firstWeekday })`

Reply:
Nice find, switched to that :+1: